tasks/php_apache.yml

```diff
@@ -1,5 +1,5 @@
   - name: LAMP | FAST-CGI configuration for php
-    template: src={{role_dir}}/templates/apache2/conf/fastcgi.conf.j2 dest=/etc/apache2/conf-available/fastcgi.conf
+    template: src={{role_dir}}/templates/apache2/conf/fastcgi-{{ php_family_prefix }}.conf.j2 dest=/etc/apache2/conf-available/fastcgi.conf
```

Comment:
If you put there php7 it doesn't work with php extension on xenial, e.g., "{{php_family_prefix}}-mysql". So I think "-php" is ok.